db/models.py

```diff
@@ -1 +1,40 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name: str = models.CharField(max_length=255, unique=True)
```

Comment:
Using type hints like 'name: str =' for Django model fields is unconventional and might lead to issues with some tools or IDEs. It's recommended to use the standard Django field definitions without type hints.